### PR TITLE
added unsplash translations

### PR DESCRIPTION
**Pull request by art-vasilyev**
## Description

Added unsplash translations.

## Checklist

- [ ] Tests (if applicable)
- [ ] Documentation (if applicable)
- [ ] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

File: ru.js [hunk 1]
```diff
@@ -141,6 +142,16 @@ l10n.translations.ru = {
       button: 'Показать'
     }
   },
+  unsplash: {
+    menu: {
+      editorial: 'Editorial',
```
Comment (homm) [inline ru.js:147]: Выбор редакции?

**Review by homm — APPROVED** [review]
For the note: github don't send any notifications on new commits, you need to leave comments either reassigning review. I've seen this PR accidentally.

**Comment by homm** [comment]
Also, tests failed.

**Comment by art-vasilyev** [comment]
> Also, tests failed.

Looks like github action failed, not a test issue: `HttpError: Resource not accessible by integration`

**Review by homm — APPROVED** [review]

**Pull request merged**